OracleWebCenterContent/dockerfiles/README.md

```diff
@@ -0,0 +1,274 @@
+Oracle WebCenter Content Container Image
+========================================
+
+## Contents
+
+### 1. [Introduction](#1-introduction-1)
+### 2. [Hardware and Software Requirements](#2-hardware-and-software-requirements-1)
+### 3. [Pre-requisites](#3-pre-requisites-1)
+### 4. [Building Oracle WebCenter Content Image](#4-building-oracle-webcenter-content-image-1)
+### 5. [Running Oracle WebCenter Content Container](#5-running-oracle-webcenter-content-container-1)
+### 6. [License](#6-license-1)
+### 7. [Copyright](#7-copyright-1)
+# 1. Introduction
+This project offers scripts to build an Oracle WebCenter Content container image based on 12.2.1.4.0 release. Use this documentation to facilitate installation, configuration, and environment setup for DevOps users. For more information about Oracle WebCenter Content, see the [Oracle WebCenter Content 12.2.1.4.0 Online Documentation](https://docs.oracle.com/en/middleware/webcenter/content/12.2.1.4/index.html).
+
+This repository includes quick start Dockerfile for WebCenter Content 12.2.1.4.0 based on Oracle Linux 7, Oracle JRE 8 (Server) and Oracle WebLogic Infrastructure 12.2.1.4.0.
```

Comment:
Please wrap Dockerfile in `` ` `` characters, i.e. `` `Dockerfile` `` so that it is rendered in a monospace font.